arduino-ide-extension/package.json

```diff
@@ -161,7 +161,11 @@
   ],
   "arduino": {
     "cli": {
-      "version": "0.29.0"
+      "version": {
+        "owner": "arduino",
+        "repo": "arduino-cli",
+        "commitish": "master"
```

Comment:
Do not use `master`. Please use a fix commit hash.

Reply:
** need to re-gen the protocols